ui/pages/permissions-connect/permissions-connect.container.js

```diff
@@ -68,18 +75,30 @@ const mapStateToProps = (state, ownProps) => {
 
   const connectPath = `${CONNECT_ROUTE}/${permissionsRequestId}`;
   const confirmPermissionPath = `${CONNECT_ROUTE}/${permissionsRequestId}${CONNECT_CONFIRM_PERMISSIONS_ROUTE}`;
+  ///: BEGIN:ONLY_INCLUDE_IN(flask)
+  const snapInstallPath = `${CONNECT_ROUTE}/${permissionsRequestId}${CONNECT_SNAP_INSTALL_ROUTE}`;
+  ///: END:ONLY_INCLUDE_IN
 
+  const totalPages = (1 + isRequestingAccounts + isSnap).toString();
```

Comment:
This crashes the UI in a normal build because `isSnap` is not defined. So we need a fence here, for the Flask and non-Flask version of this calculation. Or perhaps it would be simpler to remove the fence around the `isSnap` variable.

Reply:
Also, adding a number to a boolean is unintuitive. Maybe it would be simpler for readability to do this the more verbose way.  e.g. something like:

```
const totalPages = 1 + (isRequestingAccounts ? 1 : 0) + (isSnap ? 1 : 0);
```